resources/views/dashboard.blade.php

```diff
@@ -7,6 +7,34 @@
     <div class="container">
         <div class="row justify-content-center">
             <div class="col-md-8 col-lg-7">
+                @if(count($future) >= 1)
```

Comment:
```suggestion
                @if($future->count() >= 1)
```
...to use collections own countable method :D